test_suite/conftest.py

```diff
@@ -1,19 +1,46 @@
 import json
+
+import re
 import time
+import logging
 import pytest
-import re
-import os
 import requests
-
-from requests.adapters import HTTPAdapter
-from requests.packages.urllib3.util.retry import Retry
+import os
+import configparser
 
 from py.xml import html
 from pytest_html import extras
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
+from reportportal_client import RPLogger
 
 from lib import test_lib
 
 
+@pytest.fixture(scope="session")
+def rp_logger():
```

Comment:
According to the examples provided here https://github.com/reportportal/agent-python-pytest#readme I think that we don't need to make a second rest API call for this, we need to create markers on top of TestsAWS class and these markers will be attached as attributes to Report Portal items.
I think I'll work on this in a different PR because it might take more time probabbly , what do you think?

Reply:
Yes, it makes sense to create as separate ticket for this and another PR.
Just one comment for future PR: Let's remember we need custom attributes for each test case separately, and they won't be fixed attributes, they will be dynamic, their values will vary during runtime (e.g. AMI ID, region, instance ID, public dns, etc.).